acceptance/README.md

```diff
@@ -145,6 +145,33 @@ There also may be scenarios where you want to specify the host(s) to release. E.
     HOST_NAMES=lvwwr9tdplg351u bundle exec rake ci:release_hosts
     HOST_NAMES=lvwwr9tdplg351u,ylrqjh5l6xvym4t bundle exec rake ci:release_hosts
 
+### Running tagged tests
+
+Tests runs can be segmented by tags defined in the tests themselves. These
+tags have been standardized into the following.
+
+* risk:high
+* risk:medium
+* risk:low
```

Comment:
As it turns out, the way that Beaker implements tag filtering a test without any tags is not filtered out when using the '--tag' option facility. Therefore, untagged tests will always be run when using the '--tag' option.
@glennsarti , this behavior addresses your concern about frequently running untagged tests.
I have updated the README to reflect this.